promptulate/tools/file/toolkit.py

```diff
@@ -0,0 +1,72 @@
+import os
+from typing import List, Optional
+
+from promptulate.tools.base import BaseToolKit, Tool
+from promptulate.tools.file.tools import (
+    AppendFileTool,
+    CopyFileTool,
+    DeleteFileTool,
+    ListDirectoryTool,
+    MoveFileTool,
+    ReadFileTool,
+    WriteFileTool,
+)
+
+DefaultTools: List[Tool] = [
+    WriteFileTool,
+    AppendFileTool,
+    ReadFileTool,
+    DeleteFileTool,
+    ListDirectoryTool,
+    CopyFileTool,
+    MoveFileTool,
+]
+
+DefaultTools_str = {
+    "write": WriteFileTool,
+    "append": AppendFileTool,
+    "read": ReadFileTool,
+    "delete": DeleteFileTool,
+    "list": ListDirectoryTool,
+    "copy": CopyFileTool,
+    "move": MoveFileTool,
+}
+
+
+class FileToolKit(BaseToolKit):
+    """File ToolKit
+
+    Args:
+        root_dir: The root directory of the file tool.
+        selected_tools: The selected tools of the file tool.
+
+    Returns:
+        The instance object of the corresponding tool
+    """
+
+    def __init__(self, root_dir: str = None, modes: Optional[List[str]] = None) -> None:
```

Comment:
This is my implementation, as a reference for your optimization.

```python
class FileToolKit(BaseToolKit):
    def __init__(self, root_dir: str = None, modes: Optional[List[str]] = None) -> None:
        self.root_dir = root_dir
        self.modes = modes or []
        
        for mode in self.modes:
            if mode not in TOOL_MAPPER.keys():
                raise ValueError(
                    f"{tool} does not exist.\n"
                    f"Please select from {list(DefaultTools_str.keys())}"
                )
    
    def get_tools(self) -> List[Tool]:
        return [self.modes[mode] for mode in self.modes]
```

Reply:
[Undertone0809](https://github.com/Undertone0809) implement is better.